build.gradle

```diff
@@ -49,29 +107,61 @@ task checkstyle(type: Checkstyle) {
     source = sourceSets.main.allJava + sourceSets.test.allJava
 }
 
-task gem(type: JRubyExec, dependsOn: ["build", "gemspec", "classpath"]) {
-    jrubyArgs "-rrubygems/gem_runner", "-eGem::GemRunner.new.run(ARGV)", "build"
-    script "build/gemspec"
-    doLast { ant.move(file: "${project.name}-${project.version}.gem", todir: "pkg") }
+java {
+    withJavadocJar()
+    withSourcesJar()
+}
+
+jar {
+    from rootProject.file("LICENSE")
+}
+
+sourcesJar {
+    from rootProject.file("LICENSE")
+}
+
+javadocJar {
+    from rootProject.file("LICENSE")
 }
```

Comment:
Let's remove them at the moment.

This plugin has been published under the MIT License, and it had `LICENSE.txt`. But, in another PR, I think about switching this to the Apache License 2.0 as other `github.com/embulk` plugins do.

Reply:
Removed `from rootProject.file("LICENSE")` parts.